src/BatteryStateDisplayService.cpp

```diff
@@ -51,10 +51,11 @@ BatteryStateDisplayService::~BatteryStateDisplayService()
 void BatteryStateDisplayService::handleBatteryDataReceived(const BatteryData& batteryData)
 {
     batteryStateOfChargeService_.addData(batteryData);
-
+    batteryStateOfChargeService_.isCharging();
     QTextStream(stdout) << "Voltage: " << batteryData.voltage
         << " Current: " << batteryData.current
-        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed() << endl;
+        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed()
+        <<"Time finish charging: " <<batteryStateOfChargeService_.timeWhenChargedOrDepleted().toString()<< endl;
```

Comment:
We should have grammatically sentences.  "Time finish charging:" should be "Time until finished charging:" or something similar.